libs/features/pages/consultancy-page/src/lib/components/elewa-consultancy-about-two/elewa-consultancy-about-two.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input, OnInit} from '@angular/core';
+
+import { ImageAndText } from '@elewa-website/models/schema/ui/image-and-text';
+import { __highlightedImageData as highlightedImageData, getLayoutClass } from '@elewa-website/data/sections';
+
+@Component({
+  selector: 'elewa-website-elewa-consultancy-about-two',
+  templateUrl: './elewa-consultancy-about-two.component.html',
+  styleUrls: ['./elewa-consultancy-about-two.component.scss'],
+})
+export class ElewaConsultancyAboutTwoComponent implements OnInit {
+  @Input() imageContent !: ImageAndText[];
+  
+  ngOnInit(){
+    this.imageContent = highlightedImageData;
+  }
+
+  getLayoutClass(content: ImageAndText): string {
+    return getLayoutClass(content);
```

Comment:
The logic to get the class should be here not the in the mock data

Reply:
I have made the necessary changes